CHANGELOG.md

```diff
@@ -7,7 +7,7 @@
 - Indicate if OS-version may be frozen with '>=' prefix. ([#1945](https://github.com/getsentry/relay/pull/1945))
 - Normalize monitor slug parameters into slugs. ([#1913](https://github.com/getsentry/relay/pull/1913))
 - Smart trim loggers for Java platforms. ([#1941](https://github.com/getsentry/relay/pull/1941))
-
+- Scrub sensitive cookies. ([#1951](https://github.com/getsentry/relay/pull/1951)
```

Comment:
Please also add this entry to `py/CHANGELOG.md`. See https://github.com/getsentry/relay#instructions-for-changelogs.

Reply:
done! Thank you!